pkgs/containers/l4t.json

```diff
@@ -0,0 +1,19 @@
+[
```

Comment:
A prior version of this file and generation scripts only copied out the specific files referenced by l4t.csv from these debs. Why do we need all the contents of these debs instead of just those specific files?

Reply:
I'll double check, but I thought I remembered the space savings to not be significant between using the entire deb and using only the explicitly listed files, so this solution was simpler.

Reply:
Regardless, I moved to only use the explicitly listed files in `l4t.csv`.